bigquery_etl/cli/query.py

```diff
@@ -976,8 +981,25 @@ def _run_query(
             )
             query_stream.seek(0)
 
-            # run the query as shell command so that passed parameters can be used as is
-            subprocess.check_call(["bq"] + query_arguments, stdin=query_stream)
+            if mapped_values is None or parallelized is None:
+                # Run the query as shell command so that passed parameters can be used as is.
+                subprocess.check_call(["bq"] + query_arguments, stdin=query_stream)
+            else:
+                query_content = query_stream.read()
+                query = query_content.format(
+                    project_id=project_id,
+                    dataset_id=dataset_id,
+                    table_id=destination_table,
+                    mapped_values=mapped_values,
+                )
```

Comment:
IMO it would be better to avoid using Python string formatting syntax in queries.
* Instead of ``INSERT INTO `{project_id}.{dataset_id}.{table_id}` `` you could add `--append-table=true` and `--replace=false` parameters to the `bq query` command when parallelizing the table init. And even if `INSERT INTO {table}` is necessary, that could be added dynamically here as needed rather than requiring that template-ish code be added to the queries themselves.
* Instead of `{mapped_values}` you could configure the query parameter(s) to use in metadata, refer to those in the conditional init SQL like `@sample_id`, and pass them in to the `bq query` command.

Reply:
If you do end up still using Python string formatting syntax in queries, then `bqetl format` would need to be changed to not reformat the Python string parameters in the SQL, as it currently reformats parameters like `{foo}` to `{foo }` (with an extra space).

Reply:
Thank you. We need the insert but I removed the need for `{project_id}.{dataset_id}.{table_id}` 👍